go/internal/bertylinks/links.go

```diff
@@ -0,0 +1,45 @@
+package bertylinks
+
+import (
+	"net/url"
+
+	"google.golang.org/protobuf/proto"
+)
+
+func pseudoCodeSoPeopleCanMakeCommentInGithubReviewSystem() {
+	var (
+		typeAndVersion  int    // int that indicates bof the kind and the version, aimed to be used in a switch/case to know how to handle the rest of the URL
+		obscureBlob     string // meant to be used by code but not by human nor by "simple JS frontend"
+		additionalFlags string // optional, contains human/JS readable variables, i.e., name
+	)
+
+	// Qr format
+	qr := "https://berty.tech/id#" + typeAndVersion + "/" + obscureBlob + "/" + additionalFlags
+
+	// contact sharing
+	{
+		typeAndVersion := 1 // contact-sharing version 1
+		obscureBlob := base62.Encode(proto.Marshal(BertyID{RDVP_SEED, ACCOUNT_PK}))
```

Comment:
With padding or without ? That not clear :) (I guess without)

Reply:
What’s the best?

Reply:
Depends, for all other keys around berty we used without.
It's the same except without if nothing if found the padding is implied.
So that make shorter keys so I would go for without.

Reply:
sounds good to me :+1: